stories/index.js

```diff
@@ -30,3 +31,11 @@ storiesOf('Upvote', module)
   .add('active', () => (
     <Upvote onClick={action('clicked')} active={true} />
   ))
+
+storiesOf('Points', module)
```

Comment:
Ah - no, you've done what I meant already, I was just reading it incorrectly! :)